emails/password-reset

```diff
@@ -1,4 +1,4 @@
-Hello from Kerbal Stuff, {{user.username}}! Someone, probably you, has asked us to reset the password on your account. If that was indeed you, click this link to proceed:
+Hello from {{ site_name }}, {{user.username}}! Someone, probably you, has asked us to reset the password on your account. If that was indeed you, click this link to proceed:
```

Comment:
I think this is supposed to be "site-name".
That may have been the reason why some bits weren't working.